workalendar/europe/united_kingdom.py

```diff
@@ -34,6 +45,10 @@ def get_late_summer_bank_holiday(self, year):
             "Late Summer Bank Holiday"
         )
 
+    def non_computable_holiday(self, year):
+        non_computable = self.non_computable_holiday_dict.get(year, None)
```

Comment:
you could also default to an empty list `[]` instead of None.

Reply:
yes, it's going to be find if the test below is ``if non_computable``